### PR TITLE
Lutalo's card

### PR DESCRIPTION
**Pull request by Bindekere**
Added my card information to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
+        <div class="card">
```
Comment (TenTraicion) [inline index.html:118]:
```suggestion
        <!-- ________ Bindekere card START ________ -->
        <div class="card">
```

File: index.html [hunk 2]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
...
+          <p class="name">Bindekere Aaron</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/lutalo24" target="_blank">@average muganda</a>
```
Comment (TenTraicion) [inline index.html:122]:
```suggestion
            <a href="https://twitter.com/lutalo24" target="_blank">@lutalo24</a>
```

File: index.html [hunk 3]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.youtube.com/c/SonnySangha" target="_blank" title="As a web 3 entuhusiast Sonny Sangha's content has really improved my way of thinking when I am tackling a new project">SonnySangha - Youtube</a>
+              </li>
+              <li>
+                <a href="https://www.freecodecamp.org/" target="_blank" title="Probably the best resource overall for anyone that wants to be a developer.">FreeCodeCamp</a>
+              </li>
+            </ul>
+          </div>
+        </div>
```
Comment (TenTraicion) [inline index.html:145]:
```suggestion
        </div>
        <!-- ________ Bindekere card END ________ -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Bindekere, Thanks for your contribution. Kindly make the recommended changes and commit. Thanks!

**Comment by CandidDeer** [comment]
@Bindekere Please note that open PRs with change requests will be closed 7 days after opening if the change requests have not been resolved by then. Thx.

**Pull request closed without merging**